axelor-project/src/main/resources/views/ProjectWizard.xml

```diff
@@ -110,4 +127,8 @@
     <attribute name="refresh" expr="eval: true"
       for="project-allocation-line-action-dashlet"/>
   </action-attrs>
+  <action-method name="action-project-method-compute-employee-domain">
```

Comment:
If you need HR method's in project, you need to override the project view in HR module so override the project dashboard view in HR and add the employee dashlet etc